src/codegen/compile.ml

```diff
@@ -5324,7 +5436,9 @@ module MakeSerialization (Strm : Stream) = struct
         G.i (Binary (Wasm.Values.I32 I32Op.Or)) ^^
         get_tag ^^ compile_eq_const Tagged.(int_of_tag Array) ^^
         G.i (Binary (Wasm.Values.I32 I32Op.Or)) ^^
-        E.else_trap_with env "object_size/Mut: Unexpected tag" ^^
+        get_tag ^^ compile_eq_const Tagged.(int_of_tag Region) ^^
+        G.i (Binary (Wasm.Values.I32 I32Op.Or)) ^^
+        E.else_trap_with env "object_size/Mut: Unexpected tag." ^^
```

Comment:
@crusso This trap keeps getting triggered (see test `stable-regions-upgrade.mo`), even though I tried to add a case to detect Regions.

I was going to try to print out the tag, to see it as a number, but I didn't get that far yet.  I need to figure out how first.